### ISSUE TITLE
User needs to be able to record actions taken in an active meeting

### ISSUE THREAD

**Issue by ShaunBulbrook** [ShaunBulbrook/MinuteByMinute#14]
When a meeting is active, the user should be able to create an action by clicking a button or using a keyboard shortcut. - they should then be able to save the action in the meeting to be viewed later.

If work is started before #13 then another issue will need to be raised, to remove this functionality from views where a meeting is not active.

**Comment by ShaunBulbrook** [comment]
Blocked by #12
